site/content/en/docs/tasks/run/multikueue/plain_pods.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Run Plain Pod in Multi-Cluster"
+linkTitle: "Plain Pod"
+weight: 2
+date: 2025-02-17
+description: >
+  Run a MultiKueue scheduled Plain Pod.
+---
+
+## Before you begin
+
+1. Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+2. For the ease of setup and use we recommend using at least Kueue v0.11.0.
```

Comment:
Instead of this, could you use card like https://github.com/kubernetes-sigs/kueue/blob/8b4affc551012a8b500d2f16a12acd1e282698b5/site/content/en/docs/tasks/run/jobs.md?plain=1#L205?